src/Analyzers/VisualBasic/Analyzers/VisualBasicAnalyzersResources.resx

```diff
@@ -129,4 +129,13 @@
   <data name="If_statement_can_be_simplified" xml:space="preserve">
     <value>'If' statement can be simplified</value>
   </data>
+  <data name="Add_method_invocation_parentheses" xml:space="preserve">
+    <value>Add parentheses to method invocation</value>
```

Comment:
reosurce name and value should match.  so if the value is `Add parentheses to method invocation` then thsi should be `Add_parentheses_to_method_invocation`